src/layouts/InterfaceLayout/containers/SwapContainer/components/SwapAddressSelector/SwapAddressSelector.vue

```diff
@@ -71,12 +71,12 @@
               <p class="listed-address">
                 {{ addr.address }}
                 <!-- Address book feature
-                <span
-                  v-if="addr.address !== currentAddress && addr.currency !== 'ETH'"
-                  class="address-note"
-                  >{{ addr.currency }} {{ $t('interface.addr') }}</span
-                >
-                -->
+                                <span
```

Comment:
@SteveMieskoski are these comments necessary?

Reply:
I don't think so.  It was back when it looked like we were talking about adding an address book feature in the near future so I kept them in there, but that is a while back now.   I'm going to remove them.

Reply:
please do